src/Microsoft.AspNetCore.Mvc.Razor.Extensions/Microsoft.AspNetCore.Mvc.Razor.Extensions.csproj

```diff
@@ -11,6 +11,16 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Microsoft.AspNetCore.Razor.Language\BoundAttributeDescriptorComparer.cs">
+      <Link>BoundAttributeDescriptorComparer.cs</Link>
+    </Compile>
+  </ItemGroup>
```

Comment:
Ignore everything I just said. I was looking at the comparers but the types are equatable 👍